.github/workflows/issues.yml

```diff
@@ -1,15 +1,63 @@
 name: Add bugs to bugs project
 
-on:
+"on":
   issues:
-    types: [ opened ]
+    types: [opened, labeled]
+  issue_comment:
+    types: [created, edited]
 
 jobs:
   add-to-project:
     name: Add issue to project
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/add-to-project@v0.1.0
+      - uses: actions/add-to-project@v0.4.0
         with:
           project-url: https://github.com/orgs/timescale/projects/55
           github-token: ${{ secrets.ORG_AUTOMATION_TOKEN }}
+          labeled: bug, needs-triage
```

Comment:
I checked the existing issues in this repository. The issues are not labeled with `bug`. So, I would suggest removing lines 18 and 19 and still adding all issues to the project board.